app/(api)/api/endorsement/route.ts

```
@@ -0,0 +1,110 @@
+import { NextRequest, NextResponse } from "next/server";
+import prisma from "@/prisma/prisma";
+
+export async function GET(req: NextRequest) {
+  const { searchParams } = new URL(req.url);
+
+  // optional filters
+  const candidateId = searchParams.get("candidateId");
+  const clerkUserId = searchParams.get("clerkUserId");
+
+  // build Prisma where‑clause
+  const where: any = { hidden: false };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)